.eslintrc

```diff
@@ -1,9 +1,3 @@
 {
-  "extends": "./.eslintrc.code",
-  "env": {
-    "jest/globals": true
-  },
-  "plugins": [
-    "jest"
-  ]
+  "extends": "./config/logic/recommended.js"
```

Comment:
shoudnt we rename logic to eslint? i prefer to have the tool in there that it is clear what the config is for. if we would add for example a second tool for logic linting in the future this will be confusing.